common/client/multi_node.go

```diff
@@ -112,26 +110,22 @@ func (c *MultiNode[CHAIN_ID, RPC]) DoAll(ctx context.Context, do func(ctx contex
 			}
 		}
 		if callsCompleted == 0 {
-			err = ErroringNodeError
+			return ErroringNodeError
```

Comment:
Was this a bug? Or was this error meant to be returned after attempting the sendOnlyNodes? If the latter, then moving the check to the end would remove the ambiguity.

Reply:
cc @DylanTinianov

Reply:
It's meant to be returned after attempting the sendOnlyNodes in this case